.gitlab-ci.yml

```diff
@@ -29,7 +29,7 @@ variables:
 stages:
   - build
   - standard
-  - serpent
+  - wb
```

Comment:
shouldn't this be `wt`? I would just name these targets something like `write-through` and `write-back` to make them more readable...

Reply:
Absolutely, thanks for that.